rpc/client/interface.go

```diff
@@ -68,6 +68,9 @@ type SignClient interface {
 	BlockByHash(ctx context.Context, hash []byte) (*ctypes.ResultBlock, error)
 	BlockResults(ctx context.Context, height *int64) (*ctypes.ResultBlockResults, error)
 	Commit(ctx context.Context, height *int64) (*ctypes.ResultCommit, error)
+
+	DataCommitment(ctx context.Context, query string) (*ctypes.ResultDataCommitment, error)
+
```

Comment:
super minor: any particular reason for the empty space here?

Reply:
Because data commitment isn't related to the functions defined above and below... just to make it easy to distinguish when reading the code. I can remove it if needed.

Reply:
nah, I think its fine